tests/System.Buffers.Experimental.Tests/BufferExtensionTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.IO.Pipelines;
+using System.IO.Pipelines.Testing;
+using Xunit;
+
+namespace System.Buffers.Experimental.Tests
+{
+    public class BufferExtensionTests
```

Comment:
They are testing an extension method being added into that assembly, so that makes sense to me.